# OpenStax Chemistry, 2nd Ed. &ndash; Dean's Errata Submissions

## Chapter 6

**Execute this cell to display chemical formulas.**
It contains a LaTeX require for mhchem, which even applies to later cells.
$ \require{mhchem} $

In [1]:
# from IPython.display import display, HTML
from decimal import *
from dp import *
sources = (b, i, c, n)
import pandas as pd
pd.set_option("display.float_format", str)
def compares(f):
    return pd.DataFrame({
        'b': f(b),
        'i': f(i),
        'n': f(n),
        'c': f(c)})
# I already used c for chempy atomic weights, so speed of light sc?
sc = Decimal('2.998') * 10**8 # Speed of light (m/s)
h = Decimal('6.626') / 10**34 # Planck's constant (J s)
k = Decimal('2.179') / 10**18 # ? constant (J)

### Chapter 6 &ndash; Formulas and Concepts

$$ \begin{align}
\lambda &:= \text{wavelength} \\
\nu &:= \text{frequency (Greek letter nu)} \\
a &:= \text{amplitude (one half from trough to peak)} \\
\text{speed} &= \lambda \nu \\
c &= 2.998 \times 10^8 ms^{-1} \\
    &= \lambda\nu \text{ (for light)} \\
\\
J &= 1\ kg \frac{m^2}{s^2} \text{( note the kilograms)} \\
h &= 6.626 \times 10^{-34}\ J\ s \text{ (Planck's constant)} \\
E &:= \text{energy} \\
E &= h\nu \text{ (for one photon)} \\
    &= \frac{hc}{\lambda} \text{ (for one photon)} \\
\\
R_\infty &= \frac{k}{hc} = 1.097 \times 10^7\ m^{-1} \text{ (Rydberg constant)} \\
\frac{1}{\lambda} &= R_\infty \left(\frac{1}{n_1^2} - \frac{1}{n_2^2}\right)
    ,\ n_1 < n_2,\ n_1 \&\ n_2 \text{ integers (Rydberg equation)} \\
Z &= \text{atomic number (1 for H, 2 for He, etc.)} \\
k &= 2.179 \times 10^{-18}\ J \\
a_0 &= 5.292 \times 10^{-11}\ m \text{ (Bohr radius)} \\
\left|\Delta E\right| &= \left|E_f - E_i\right| = h\nu = \frac{hc}{\lambda} \\
\Delta E &= k \left( \frac{1}{n_1^2} - \frac{1}{n_2^2} \right) = \frac{hc}{\lambda} \\
E_n &= - \frac{kZ^2}{n^2} \text{ (orbital energies for atoms/ions with 1 electron)} \\
r &= \frac{n^2}{Z} a_0 \text{ (radius for same)} \\
E &= 0 \text{ is the ionization limit where the electron is freed} \\
\Delta E &= E_{n\to\infty} - E_1 = 0 + k = k \text{ (ionization energy from n=1)} \\
\\
\nu &:= \text{frequency} \\
v &:= \text{velocity} \\
p &:= \text{momentum} \\
\lambda &= \frac{h}{p} = \frac{h}{mv} \text{ (de Broglie wavelength for particles)} \\
\end{align} $$

### Chapter 6 &ndash; Examples

#### Example 6.1 - Solution

In [2]:
sc / (Decimal('589') / 10**9)

Decimal('508998302207130.7300509337861')

$$ \begin{align}
c &= \lambda \nu \\
\nu &= \frac{c}{\lambda} \\
\nu &= 5.09 \times 10^{14}\ s^{-1} \\
\end{align} $$

#### Example 6.1 - Check Your Learning

In [3]:
sc / (Decimal('850') * 10**6)

Decimal('0.3527058823529411764705882353')

$$ \lambda = \frac{c}{\nu} $$
$$ \lambda = 0.353\ m $$

#### Example 6.2 - Solution

In [4]:
lambda_Ne = Decimal('640') / 10**9
nu_Ne = sc / lambda_Ne
E_photon = h * nu_Ne
E_photon

Decimal('3.1038668750000E-19')

$$ E = 3.10 \times 10^{-19} J $$

#### Example 6.2 - Check Your Learning

In [5]:
nu_micro = 3 * 10**9
E_photon = h * nu_micro
E_photon

Decimal('1.9878000000000E-24')

$$ E = 2 \times 10^{-24} J $$

#### Example 6.3 - Solution
(a) False, does not change
(b) False, frequency
(c) True
(d) False, brightness or intensity

**Note:** I got (d) mostly wrong.  The book answer:
> (d) True with regard to the threshold energy binding the electrons to the metal. Below this threshold, electrons are not emitted and above it they are. Once over the threshold value, further increasing the frequency does not increase the number of ejected electrons

#### Example 6.3 - Check Your Learning

In [6]:
nu_Al_thresh = Decimal('9.87') * 10**14 # Hz
E_phot_Al_thresh = h * nu_Al_thresh
display(E_phot_Al_thresh)
# Z_Al = 13
E_thresh_per_mole = E_phot_Al_thresh * Avo
display(E_thresh_per_mole / 10**3)

Decimal('6.53986200000000000000E-19')

Decimal('393.8304896400000000000000')

$$ E_{threshold} = 394\ kJ/mol $$
<strong style="color:blue">Huh?</strong> I got the right answer, but I have no idea why you would multiply the photon energy by a mole of electrons.

#### Example 6.4 - Solution
If a spark promotes the electron in a hydrogen atom into an orbit with n = 3, what is the calculated energy, in joules, of the electron?

In [7]:
-k / 3**2

Decimal('-2.421111111111111111111111111E-19')

$$ \begin{align}
E_n &= \frac{-kZ^2}{n^2} = \frac{k}{n^2} \text{ (Z=1 for hydrogen)} \\
    &= - 2.421 \times 10^{-19}\ J \\
\end{align} $$
**Oops:** I originally forgot the negative.

#### Example 6.4 - Check Your Learning
[Essentially asks what about n = 6.]

In [8]:
-k / 6**2

Decimal('-6.052777777777777777777777778E-20')

$$ E_6 = -6.053 \times 10^{-20}\ J $$

#### Example 6.5 - Solution
What is the energy (in joules) and the wavelength (in meters) of the line in the spectrum of hydrogen that represents the movement of an electron from Bohr orbit with n = 4 to the orbit with n = 6? In what part of the electromagnetic spectrum do we find this radiation?

In [9]:
E_4 = - k / 4**2
E_6 = - k / 6**2
E_4to6 = E_6 - E_4
E_4to6_b = float(k) * ( 1/4**2 - 1/6**2)
l = h * sc / E_4to6
display(E_4, E_6, E_4to6, E_4to6_b, l)

Decimal('-1.361875E-19')

Decimal('-6.052777777777777777777777778E-20')

Decimal('7.565972222222222222222222222E-20')

7.565972222222223e-20

Decimal('0.000002625538055988985773290500230')

$$ \Delta E = 7.566 \times 10^{-20}\ J $$
$$ \lambda = 2.626 \times 10^{-6}\ m \text{ (in near infrared)}$$
**Oops:** I fat fingered a digit at first.

#### Example 6.5 - Check Your Learning
What is the energy in joules and the wavelength in meters of the photon produced when an electron falls from the n = 5 to the n = 3 level in a He+ ion (Z = 2 for He+)?

In [10]:
E_5 = -k * 2**2 / 5**2
E_3 = -k * 2**2 / 3**2
E_5to3 = E_3 - E_5
E_5to3_b = float(k) * (2**2 / 5**2 - 2**2 / 3**2)
l = h * sc / E_5to3
display(E_5, E_3, E_5to3, E_5to3_b, l)

Decimal('-3.4864E-19')

Decimal('-9.684444444444444444444444444E-19')

Decimal('-6.198044444444444444444444444E-19')

-6.1980444444444435e-19

Decimal('-3.205002509752179899036255163E-7')

$$ \Delta E = -6.198 \times 10^{-19}\ J $$
$$ \lambda = -3.205 \times 10^{-7}\ m $$
**Oops:** I messed up both signs by not reading the question carefully enough.

#### Example 6.6 - Solution
If an electron travels at a velocity of 1.000 × 107 m s–1 and has a mass of 9.109 × 10–28 g, what is its wavelength? 

In [12]:
v = Decimal('1.000') * 10**7
m_e = Decimal('9.109') / 10**28 / 10**3
p = v * m_e
l = h / p
display(v, m_e, p, l)

Decimal('10000000.000')

Decimal('9.109E-31')

Decimal('9.1090000000000E-24')

Decimal('7.274124492260401800417169832E-11')

$$ \lambda = 7.274 \times 10^{-14}\ m $$
**Oops:** I forgot to convert units since Joules involves kilograms, not grams.
$$ \lambda = 7.274 \times 10^{-11}\ m $$

#### Example 6.6 - Check Your Learning
Calculate the wavelength of a softball with a mass of 100 g traveling at a velocity of 35 m s–1, assuming that it can be modeled as a single particle.

In [14]:
m_ball = Decimal('100') / 10**3
v_ball = Decimal('35')
l = h / (m_ball * v_ball)
l

Decimal('1.893142857142857142857142857E-34')

$$ \lambda = 1.9 \times 10^{-34} $$
**Oops:** I messed up my calculations (didn't group a divide in Python).  I need to start being more careful.

### Chapter 6 &ndash; Exercises